raiden/tests/utils/smartcontracts.py

```diff
@@ -54,7 +56,7 @@ def deploy_contract_web3(
         contract_name: str,
         deploy_client: JSONRPCClient,
         num_confirmations: int = None,
-        *args,
+        constructor_arguments: typing.Tuple[typing.Any] = (),
```

Comment:
this type is not correct, the tuple is not necessarily of length `1`. The correct syntax is [`typing.Tuple[typing.Any, ...]`](https://mypy.readthedocs.io/en/latest/builtin_types.html?highlight=tuple#built-in-types)